src/scales/scale.time.js

```diff
@@ -807,8 +807,8 @@ module.exports = Scale.extend({
 		var size = me._getLabelSize(exampleLabel);
 		var capacity = Math.floor(me.isHorizontal() ? me.width / size.w : me.height / size.h);
 
-		if (me.options.offset) {
-			capacity--;
+		if (!me.options.offset) {
+			capacity++;
```

Comment:
I don't think this change is correct because `capacity` is used as the number of tick intervals in `determineStepSize` and `determineUnitForAutoTicks`, not as the number of ticks.

Reply:
You're right. I'll close this PR and take care of the `_autoSkip` part in the PR where I'm adding major tick support